src/pybricks/geometry.pyi

```diff
@@ -5,12 +5,16 @@ from _typeshed import NoneType
 from typing import Collection, Optional, Tuple
 
 class Matrix:
-    def __init__(self, rows: Collection[int]): ...
+    def __init__(self, rows: Collection[Collection[int]]): ...
     @property
     def T(self) -> Matrix: ...
     @property
     def shape(self) -> Tuple[int, int]: ...
 
+    def __add__(self, other) -> Matrix: ...
+    def __sub__(self, other) -> Matrix: ...
+    def __mul__(self, other) -> Matrix: ...
```

Comment:
If we want to be very precise, we should add `/`, `//`, and `rmul` as well.

Reply:
Wondering if `__rmul__`, `__radd__` etc. have any sense.

If matrix calculation is called (e.g. `matrix1 * matrix2`, or `matrix * scalar`), the `__mul__` will be used.
When a matrix is on the right side of multiplication, and the left side does not implement `__mul__`, we don't want to allow such operations anyway.

Reply:
`rmul` is used for `scalar * Matrix`.

In-place division or multiplication by a scalar is also allowed.

If you are curious, the source is [here](https://github.com/pybricks/pybricks-micropython/blob/7e269968652eb3dfc00990596dfc7a827574f996/pybricks/geometry/pb_type_matrix.c#L380-L408).

Reply:
Silly me. Did not realise `scalar * Matrix` is allowed.

All missing operations should be there now.